common/src/main/scala/explore/components/Tile.scala

```diff
@@ -135,31 +135,22 @@ object Tile:
             ^.key := "tile-${p.id.value}"
           )(
             // Tile title
-            <.div(ExploreStyles.TileTitle)(
-              ^.key := s"tileTitle-${p.id.value}",
+            <.div(ExploreStyles.TileTitle, ^.key := s"tileTitle-${p.id.value}")(
               // Title and optional back button
-              <.div(
-                ExploreStyles.TileTitleMenu |+| p.tileTitleClass,
+              <.div(ExploreStyles.TileTitleMenu |+| p.tileTitleClass)(
                 p.back.map(b => <.div(ExploreStyles.TileButton, b)),
```

Comment:
is it correct to move the back  button to the title text?

Reply:
This didn't change. I just separated style and content into two different parameter lists (which I find it resembles HTML's structure of attributes together with the tag and then content in a different block).